test/opf/sparse_sdpwrm.jl

```diff
@@ -0,0 +1,59 @@
+using LinearAlgebra
+using SparseArrays
+
+function test_opf_pm(::Type{PGLearn.SparseSDPOPF}, data::Dict)
+    OPF = PGLearn.SparseSDPOPF
+
+    data["basic_network"] || error("Input data must be in basic format to test")
+    N = length(data["bus"])
+    E = length(data["branch"])
+    G = length(data["gen"])
+
+    # Solve OPF with PowerModels
+    solver = OPT_SOLVERS[OPF]
+    res_pm = PM.solve_opf(data, PM.SparseSDPWRMPowerModel, solver)
+
+    # Build and solve OPF with PGLearn
+    solver = OPT_SOLVERS[OPF]
+    opf = PGLearn.build_opf(OPF, data, solver)
+    PGLearn.solve!(opf)
+    res = PGLearn.extract_result(opf)
+
+    # Check that the right problem was indeed solved
+    @test res["meta"]["formulation"] == string(OPF)
+    @test res["meta"]["termination_status"] ∈ ["OPTIMAL", "ALMOST_OPTIMAL"]
+    @test res["meta"]["primal_status"] ∈ ["FEASIBLE_POINT", "NEARLY_FEASIBLE_POINT"]
+    @test res["meta"]["dual_status"] ∈ ["FEASIBLE_POINT", "NEARLY_FEASIBLE_POINT"]
+    # Check objective value against PowerModels
+    @test isapprox(res["meta"]["primal_objective_value"], res_pm["objective"], atol=1e-3, rtol=1e-3)
+    @test isapprox(res["meta"]["primal_objective_value"], res["meta"]["dual_objective_value"], rtol=1e-6)
```

Comment:
Those tolerances are quite loose on the primal side, can we make them `1e-4` instead?